lib/api/manipulation.js

```diff
@@ -363,6 +363,47 @@ exports.wrapInner = _wrap(function (el, elInsertLocation, wrapperDom) {
   updateDOM(wrapperDom, el);
 });
 
+/**
+ * The .unwrap() function, removes the parents of the set of matched elements
+ * from the DOM, leaving the matched elements in their place.
+ *
+ * @example <caption>without selector</caption>
+ *   const $ = cheerio.load(
+ *     '<div id=test>\n  <p>Hello</p></div>\n  <div><p>World</p>\n</div>'
```

Comment:
This example doesn't seem entirely correct. Why wouldn't `#test` be deleted?

Reply:
You are correct and I updated example, sorry. Second one is correct.